composer.json

```diff
@@ -4,9 +4,9 @@
     "type": "symfony-bundle",
     "license": "MIT",
     "require": {
-        "php": "8.*",
+        "php": ">=8.1",
```

Comment:
I'm against such notation, this in fact promises the lib will work with PHP 9, 10, 15, and whatnot. We have no idea whether that's true